_data/master/navbars/usage.yml

```diff
@@ -29,6 +29,8 @@ toc:
   path: /usage/configuration/ip-in-ip
 - title: Configuring a Node IP Address and Subnet
```

Comment:
I wonder if as we add more usage docs it will make sense to put these within a top-level `Managing and Configuring Nodes` section.  Probably don't need to do it now.